templates-java/pom.xml

```diff
@@ -5,11 +5,23 @@
     <parent>
         <artifactId>spring-boot-cli</artifactId>
         <groupId>br.com.netodevel</groupId>
-        <version>0.0.1.BUILD-SNAPSHOT</version>
+        <version>0.0.2.BUILD-SNAPSHOT</version>
```

Comment:
The version `0.0.2.BUILD-SNAPSHOT` may indicate a development or unstable build. Consider using a more specific version or a stable release.